cost-analyzer/charts/grafana/templates/role.yaml

```diff
@@ -1,6 +1,6 @@
 {{ if .Values.global.grafana.enabled }}
 {{- if .Values.rbac.create }}
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Will changes like this cause problems on older versions of K8s? I know we technically support quite old versions.

Reply:
Or, to phrase this another way, when did rbac go from `v1beta1` to `v1`?

Reply:
In this particular case, no, since we don't switch on v1beta for rbac.authorization.k8s.io anywhere else in the chart.